pipelines/build-deploy-test-CI-WL.yml

```diff
@@ -2715,6 +2899,47 @@ jobs:
         RABBITMQ_CONFIG_VALUES_FILE: rabbitmq/values.yml
       input_mapping: {census-rm-kubernetes-dependencies-repo: census-rm-kubernetes-dependencies-repo}
 
+# Apply Database Patches
+- name: "WL Apply Database Patches"
+  serial: true
+  serial_groups: [wl-database-patches,
+                  wl-terraform,
+                  wl-action-scheduler,
+                  wl-action-worker,
+                  wl-case-api,
+                  wl-case-processor,
+                  wl-uac-qid-service,
+                  wl-pubsub,
+                  wl-print-file-service,
+                  wl-fieldwork-adapter,
+                  wl-notify-processor,
+                  wl-exception-manager,
+                  wl-toolbox,
+                  wl-database-monitor,
+                  wl-rabbitmonitor,
+                  wl-regional-counts]
+  on_failure: *slack_failure_alert_wl
+  on_error: *slack_error_alert_wl
+  plan:
+  - get: census-rm-kubernetes-dependencies-repo
+  - get: census-rm-kubernetes-repo
```

Comment:
Could use existing census-rm-kubernetes-repo instead

Reply:
Changed to `census-rm-kubernetes-optional-repo` instead